snapraid-runner.py

```diff
@@ -52,9 +52,14 @@ def snapraid_command(command, args={}, ignore_errors=False):
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE)
     out = []
-    threads = [
-        tee_log(p.stdout, out, logging.OUTPUT),
-        tee_log(p.stderr, [], logging.OUTERR)]
+    if command == 'smart':
+        threads = [
+            tee_log(p.stdout, out, logging.SMART),
+            tee_log(p.stderr, [], logging.OUTERR)]
+    else:
+        threads = [
+            tee_log(p.stdout, out, logging.OUTPUT),
+            tee_log(p.stderr, [], logging.OUTERR)]
```

Comment:
Which output belongs to which command is always clearly identifiable from the headers. Why does `smart` need this logging trickery?

Reply:
I created a new SMART logging level that is output in both "short" and "long" logging modes since it is highly likely that it is always desired to view the SMART output if you are sending emails on success. It does not output if set to only error notification emails since the log level is set one level higher than error logging. This section ensures the SMART "OUTPUT" logging is redirected into "SMART" logging while leaving all other logging as-is.

Reply:
Ah, ok – that makes sense. Please add a comment about that to where you define that loglevel.
Also add an `output_log_level` parameter to this function and use that instead of this `if`.

Reply:
I added a comment on line 175 to highlight the SMART logging level creation, but I'm not sure what you're looking for in the `output_log_level` instead of this `if`

Reply:
Thanks for the changes. I meant that the signature should be `snapraid_command(command, args={}, ignore_errors=False, output_log_level=logging.OUTPUT)` and then use the parameter instead of guessing based on the name of the command.